src/main/java/wiresegal/wob/TokenHolder.kt

```diff
@@ -23,6 +23,7 @@ private val gitProperties: Properties by lazy {
 
 val token: String? = System.getenv("DISCORD_TOKEN")
 val arcanumToken: String? = System.getenv("ARCANUM_TOKEN")
+val homepageTarget = System.getenv("BRANDONSANDERSON_URL") ?: "https://www.brandonsanderson.com/"
```

Comment:
I am probably running one of the few versions of this bot for the Will Wight discord group and this tools would obviously not work so well for other authors sites.  Sadly I have 0 familiarity with Koltin so it would take a significant lift to learn in order to do a PR, is it possible to NOT include a default URL and add a test for `homepageTarget` being defined before registering / enabling the progress command?  If not I'll dig in and see if I can suggest a different change.

Reply:
Sure, that's very reasonable request. I hope Ole will find some time to implement that :)

Reply:
I'll get this done tomorrow, should be quick!

Reply:
Fixed in pull request #7

Reply:
Thank you very much!